app/assets/stylesheets/components/_mobile-filters.scss

```diff
@@ -109,6 +114,28 @@
     .facets__tags-block {
       display: block;
     }
+
+    // Temporary spacing fix because first item is shorter than the others
+    .facets__content > .app-c-expander:first-child {
+      padding-top: govuk-spacing(2);
+    }
+
+    // Temporary override for consistent heading sizes. Needs manual
+    // intervention in the future to determine correct approach after
+    // the design system font size changes have been rolled out.
+    .result-info__header .gem-c-heading {
+      @include govuk-font($size: 19, $weight: 700);
+    }
+
+  }
+}
+
+// The vertical position of other expander icons was fixed in _expander.scss
+// as it's app specifc. However, we also have gem expanders mixed in here
+// so these need the same override, for all breakpoints:
+.js-enabled {
+  .facets__content .gem-c-option-select__icon {
```

Comment:
This should be fixed in the option select component in the gem.

Reply:
This has been updated in the gem as requested, see here: https://github.com/alphagov/govuk_publishing_components/pull/4256